### DISCUSSION TITLE
Remove/Hide My Calendars from Calendar panel

### DISCUSSION THREAD

**Discussion by arsaboo** [home-assistant/frontend#11261] [Other feature requests]
## The request
The current calendar panel lists all the calendars in the account. While this is useful at the time of configuration, we don't need to display it all the time. Can we hide it under some configuration option? 
![image](https://user-images.githubusercontent.com/18319734/81868199-f559c200-953f-11ea-8277-74b8a1c7c16b.png)

**Comment by andriej** [comment]
You can set parameters to 'false' in yaml file, at least for google calendar (and not track them)

**Comment by koles142** [comment]
The same, I'm using calendar for automation. I have to track them but I won't see this panel in frontend. How can I do this?

**Comment by noxhirsch** [comment]
I also really need a way to permanently hide calendars from the list on the left and the view on the right. In total I have 26 calendars - five useful and the rest are todo-lists and calendars for automation.

**Comment by bramkragten** [comment]
In 0.115 you can hide panels, and add a calendar Lovelace panel for which you can select what entities to use.

**Reply by DEADSEC-SECURITY** [reply]
How? I can hide dashboards but not the history/map etc menus

**Comment by zsarnett** [comment]
We should store this using the new Local Storage Decorator that Bram created. Storing only the not checked calendars

**Comment by DEADSEC-SECURITY** [comment]
I opened a [feature request](https://github.com/home-assistant/frontend/discussions/13818) in the same area but more generalized for other panels too.

**Comment by loopy321** [comment]
Is there a solution for this?  I have the same issue as OP.

**Reply by MagicCheese1** [reply]
Workaround: https://www.reddit.com/r/homeassistant/comments/1gonlmd/comment/m5d0n0l/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button
